src/Razor/test/Microsoft.AspNetCore.Razor.Test.Common/Language/TestProject.cs

```diff
@@ -8,15 +8,35 @@ namespace Microsoft.AspNetCore.Razor.Language
 {
     public static class TestProject
     {
+        public static string GetProjectDirectory(string directoryHint)
```

Comment:
This logic is an anti-pattern but I'm not blocking the PR in order to clean this up, so I'm rolling with it.

Reply:
File an issue if this is about more than enabling this project to be runnable in Helix.

Reply:
It's mostly that.

Reply:
https://github.com/dotnet/aspnetcore/issues/22100